build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ls -op packer | grep -v / | awk '{print $8}' | tail -n +2 | while read image
+
+do
+    packer build packer/$image
+done
```

Comment:
We want to run all of the packer images. If I'm not mistaken this only runs 1? but I will use this in the future.